### PR TITLE
Feature/atomic predicates

### PR DESCRIPTION
**Pull request by tkmct**
Atomic predicates
- Equal
- IsLessThan
- IsSameAmount
- IsContained

### CODE REVIEW

File: __tests__/ovm/deciders/EqualDecider.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { initializeDeciderManager } from '../helpers/initiateDeciderManager'
+
+describe('EqualDecider', () => {
+  const deciderManager = initializeDeciderManager()
+  const EqualDeciderAddress = deciderManager.getDeciderAddress(
+    AtomicPredicate.Equal
+  )
+
+  test('decide true', async () => {
+    const property = new Property(EqualDeciderAddress, [
```
Comment (syuhei176) [inline __tests__/ovm/deciders/EqualDecider.test.ts]: Great! New spec of 4 deciders are correct.
BTW, I was also thinking how do we write test code for atomic decider.
atomic deciders such as EqualDecider or IsContainedDecider don't depend on DeciderManager, but it requires in argument of `decide` method. We can use EqualDecider directly, but we need mock.
So, it seems that 1.using mock DeciderManager or 2. using another interface of atomic decider(which doesn't depends on DeciderManager) are better options. What do you think?

Comment (tkmct) [reply __tests__/ovm/deciders/EqualDecider.test.ts]: I think call decide method directly is the simplest way.
I'll fix this.

Comment (tkmct) [reply __tests__/ovm/deciders/EqualDecider.test.ts]: I guess 1 is okay for now. I just fixed the test. please review.

File: src/ovm/deciders/atomic/IsContainedDecider.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * first range is contained by second range: first.start >= second.start && first.end <= second.end
+ */
+export class IsContainedDecider implements Decider {
+  public async decide(
+    manager: DeciderManagerInterface,
+    inputs: Bytes[]
+  ): Promise<Decision> {
+    if (inputs.length !== 2) {
+      return {
+        outcome: false,
```
Comment (syuhei176) [inline src/ovm/deciders/atomic/IsContainedDecider.ts]: Let's discuss whether we should return false or throw exception(It means undecided maybe) later!

Comment (tkmct) [reply src/ovm/deciders/atomic/IsContainedDecider.ts]: sure!

**Pull request merged**